core/templates/dev/head/pages/exploration-player-page/services/extract-image-filenames-from-state.service.ts

```diff
@@ -155,8 +156,11 @@ oppia.factory('ExtractImageFilenamesFromStateService', [
       }
       var allHtmlOfState = _getAllHtmlOfState(state);
       allHtmlOfState.forEach(function(htmlStr) {
-        filenamesInState = filenamesInState.concat(
-          _extractFilepathValueFromOppiaNonInteractiveImageTag(htmlStr));
+        imageIdsInState = imageIdsInState.concat(
+          _extractImageIdValueFromOppiaNonInteractiveImageTag(htmlStr));
+      });
+      imageIdsInState.forEach(function(imageId) {
+        filenamesInState.push(state.imageAssets.imageMapping[imageId].src);
```

Comment:
Can we add a method in ImageAssets object `get_image_src(image_id)`?

Reply:
Yes we can, but is there any issue with this?